usr/share/rear/restore/BAREOS/default/400_restore_backup_bextract.sh

```diff
@@ -0,0 +1,75 @@
+#
+# Restore from Bareos using bextract (optional)
+#
+
+### Create example bootstrap file
+cat <<EOF >$VAR_DIR/bootstrap.txt
+### Example Bareos bootstrap file
+###
+
+### Only the exact Volume name is required, other keywords are optional.
+### The use of FileIndex and Count keywords speeds up the selection enormously.
+
+### The Volume name to use
+Volume=PLEASE-EDIT-BOOTSTRAP
+
+### A (list of) Client name(s) to be matched on the current Volume
+#Client=$(hostname -s)-fd
+
+### A (list or range of) JobId(s) to be selected from the current Volume
+#JobId=18
+
+### A (list of) Job name(s) to be matched on the current Volume
+#Job=Bkp_Daily.2011-06-16
+
+### A (list or range of) Volume session id(s) to be matched from the current Volume
+#VolSessionId=1
+
+### The Volume session time to be matched from the current Volume
+#VolSessionTime=108927638
+
+### A (list or range of) FileIndex(es) to be selected from the current Volume
+#FileIndex=1-157
+
+### The total number of files that will be restored for this Volume.
+#Count=157
+EOF
+#############################################################################
+
+if [[ "$BEXTRACT_DEVICE" || "$BEXTRACT_VOLUME" ]]; then
+
+    if [[ -s "$TMP_DIR/restore-exclude-list.txt" ]]; then
+        exclude_list=" -e $TMP_DIR/restore-exclude-list.txt "
+    fi
+
+    if [[ -b "$BEXTRACT_DEVICE" && -d "/backup" ]]; then
+
+        ### Bareos support using bextract and disk archive
+        LogPrint "
+The system is now ready to restore from Bareos. bextract will be started for
+you to restore the required files. It's assumed that you know what is
+necessary to restore - typically it will be a full backup.
+
+Do not exit 'bextract' until all files are restored.
+
+WARNING: The new root is mounted under '$TARGET_FS_ROOT'.
```

Comment:
Is the word "WARNING" really needed here?
Cf.
https://schlomo.schapiro.org/2015/04/warning-is-waste-of-my-time.html

And what is "the new root"?
Was there some "the old root" that is now gone?
And what had that "the old root" been?
;-)

Cf. our "unagitated" wording in
usr/share/rear/wrapup/default/980_good_bye.sh
```
The target system is mounted at '$TARGET_FS_ROOT'.
```